cpp/src/gandiva/gdv_function_stubs_test.cc

```diff
@@ -949,4 +949,65 @@ TEST(TestGdvFnStubs, TestMaskLastN) {
   EXPECT_EQ(expected, std::string(result, out_len));
 }
 
+TEST(TestGdvFnStubs, TestConv) {
+  gandiva::ExecutionContext ctx;
+
+  int64_t ctx_ptr = reinterpret_cast<int64_t>(&ctx);
+  gdv_int32 out_len = 0;
+
+  const char* value = conv_utf8_int32_int32(ctx_ptr, "1000101101", 10, 2, 10, &out_len);
```

Comment:
If the user defines a number with many zeros at the left of the string, the function will work fine?

Example: "000000000001" in binary to decimal -> "1"
Other example: "-000000001" in binary to decimal -> "-1"

Reply:
Tested, it is ok, I added new tests for this.